test/benchmark/tests/utils/__init__.py

```diff
@@ -27,8 +28,9 @@
 GPU_INSTANCES = ["p2", "p3", "p4", "g2", "g3", "g4"]
 
 # DLAMI with nVidia Driver ver. 450.119.03 (support upto CUDA 11.2), Ubuntu 18.04
```

Comment:
Is this description still valid for the new AMI?

Reply:
Yes, that it still holds true.